locales/en-US/addon.properties

```diff
@@ -6,8 +6,11 @@ tooltip_send_to_tab= Send to tab
 tooltip_minimize= Minimize
 tooltip_maximize= Maximize
 tooltip_close= Close
+tooltip_switch_visual= Click to switch visualization
+media_type_audio= audio
+media_type_video= video
 error_msg= Something's gone wrong with this video. Try again later.
 error_link= Open in new tab
 # LOCALIZATION NOTE: placeholder is domain name
-loading_msg= Loading video from %s
-content_menu_msg= Send to mini player
\ No newline at end of file
+loading_msg= Loading %s from %s
```

Comment:
you'll need to update the localization note to make clear which placeholder is which